packages/openapi-fetch/test/index.test.ts

```diff
@@ -1843,6 +1830,41 @@ describe("client", () => {
       );
     });
   });
+
+  describe("URL as property style call", () => {
+    it("performs a call without params", async () => {
+      const client = createClient<paths>({ baseUrl });
+      const { getRequest } = useMockRequestHandler({
+        baseUrl,
+        method: "get",
+        path: "/anyMethod",
+      });
+      await client["/anyMethod"].GET();
+      expect(getRequest().method).toBe("GET");
+    });
+
+    it("performs a call with params", async () => {
+      const client = createClient<paths>({ baseUrl });
+      const { getRequestUrl } = useMockRequestHandler({
+        baseUrl,
+        method: "get",
+        path: "/blogposts/:post_id",
+        status: 200,
+        body: { message: "OK" },
+      });
+
+      await client["/blogposts/{post_id}"].GET({
+        // expect error on number instead of string.
+        // @ts-expect-error
+        params: { path: { post_id: 1234 } },
+      });
+
+      await client["/blogposts/{post_id}"].GET({
+        params: { path: { post_id: "1234" } },
+      });
+      expect(getRequestUrl().pathname).toBe("/blogposts/1234");
```

Comment:
Comment: these are great tests, by the way! 🙂 I would also like to test this API for `POST` calls and fetches with requestbodies in addition to what you have here. But this is a great start.

Reply:
I have added some more. They are all adapted from the base tests above.

I'm a bit unsure how much we want to cover this API: On one hand, it is a new public API, so we should probably err on the "too-many-tests-side". On the other hand, the integration is quite thin, so a lot of the core type inference and calling logic is re-used. So I'm not sure it makes sense to essentially test it twice.

WDYT?